genesis-simple-hooks.php

```diff
@@ -1,6 +1,13 @@
 <?php
 /**
- * This file handles the creation of the Simple Hooks admin menu.
+ * Plugin Name: Genesis Simple Hooks
+ * Plugin URI: https://studiopress.com
+ * Description: Creates a new Genesis settings page that allows you to insert code (HTML, Shortcodes, and PHP), and attach it to any of the 50+ action hooks throughout the Genesis Theme Framework, from StudioPress.
+ * Author: StudioPress
+ * Author URI: https://www.studiopress.com/
+ * Version: 2.3.1
+ * License: GPL2+
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
```

Comment:
This was missing, and caused an error on installing from wp.org:

<img width="1066" alt="Screenshot 2023-09-29 at 12 13 31 PM" src="https://github.com/studiopress/genesis-simple-hooks/assets/4063887/b188071d-9147-4c34-a536-ab43e52427d0">